lib/plugin-loader.js

```diff
@@ -427,6 +427,16 @@ NodeAuthenticationPlugIn.prototype = {
     }
     return true;
   },
+
+  isRequestedAuthPlugin(context) {
+    //we should not load authentication types that are 
+    //not requested by the administrator
+    if (!context.authManager.authPluginRequested(this.identifier,
```

Comment:
if the value of the function return is equal to the value that you are returning, then you can just return the function itself, as `return context.authManager.authPluginRequested(this.identifier, this.authenticationCategory)`, or alternatively since it is a one-liner at that point, that this can be done inline.